x-pack/plugins/alerting/server/task_runner/fixtures.ts

```diff
@@ -99,6 +99,7 @@ export const generateSavedObjectParams = ({
         last_run: {
           timestamp: '1970-01-01T00:00:00.000Z',
           metrics: {
+            duration: 0,
```

Comment:
@XavierM I had to fix some false-negative failures of some of the existing tests because they assert too much, i.e. last-run metrics in this case. None of them were actually about metrics. Using [`expect.objectContaining`](https://jest-bot.github.io/jest/docs/expect.html#expectobjectcontainingobject) can help make tests more resilient. @xcrzx described that problem perfectly in https://github.com/elastic/security-team/issues/3526#issuecomment-1096724323